source/MaterialXGraphEditor/RenderView.cpp

```diff
@@ -329,70 +322,6 @@ void RenderView::loadMesh(const mx::FilePath& filename)
     }
 }
 
-void RenderView::loadDocument(const mx::FilePath& filename, mx::DocumentPtr libraries)
```

Comment:
Mostly just an unnecessary duplicate of Graph load. validate() check added to Graph version to make them identical.